server/webapp/WEB-INF/rails.new/app/helpers/api/users_helper.rb

```diff
@@ -0,0 +1,41 @@
+##########################################################################
+# Copyright 2017 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module Api
+  module UsersHelper
```

Comment:
We intentionally do not use shared helpers between API versions. If you need the same helper in multiple API versions, please copy-paste code.

This makes it easy to change one api without affecting other api versions in any manner.